sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -571,7 +571,7 @@ export function fromAmqpMessage(
  * Describes the message received from Service Bus.
  * @interface ReceivedMessage
  */
-interface ReceivedMessage extends ReceivedMessageInfo {
+export interface ReceivedMessage extends ReceivedMessageInfo {
```

Comment:
As discussed before, can we undo the changes in this file in the spirit of not doing any code fixes?